tests/integration/test_sanity.py

```diff
@@ -189,6 +189,32 @@ def test_readonly_commands(cluster):
     assert cluster.node(1).client.get('key') == b'value'
 
 
+def test_readonly_acl_oom_permutations(cluster):
+    cluster.create(3)
+
+    cluster.execute('set', 'abc', 1234)
+    cluster.wait_for_unanimity()
+
+    # no oom, acl allow
+    assert cluster.execute("get", "abc") == b'1234'
+
+    # oom, acl allow
+    cluster.execute('config', 'set', 'maxmemory', 1)
+    assert cluster.execute("get", "abc") == b'1234'
+    cluster.execute('config', 'set', 'maxmemory', 0)
```

Comment:
```suggestion
    cluster.config_set('maxmemory', 0)
```
I don't expect anything different but maybe we should use this function to set config on all nodes? Please consider it for other usages as well.

Reply:
ok

Reply:
Are you going to make this change? forgot to push the new commit?

Reply:
Ive combined this new test (as it was essentially the same as the scrip test into the scrip test, just adding not script test cases.  did this change at same time